src/joint.dia.element.js

```diff
@@ -694,8 +694,6 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
     pointerdown: function(evt, x, y) {
 
-        this.model.trigger('batch:start');
-
         // target is a valid magnet start linking
         if (evt.target.getAttribute('magnet') && this.paper.options.validateMagnet.call(this.paper, this, evt.target)) {
 
```

Comment:
The problem I am trying to solve with the batch:start and batch:stop is the double call, once in the derived class  and once in the base class (ElementView.pointerdown   / CellView.pointerdown).   Since the base class call is essentially inlined (by applying this) the event is triggered twice on the same model- e.g. this.model.trigger('batch:start') where this.model is Element.   Since we are always invoking the base class event, it seems correct to just handle it there.